testhelper/docker/resource/minio/minio.go

```diff
@@ -116,3 +126,46 @@ func Setup(pool *dockertest.Pool, d resource.Cleaner, opts ...func(*Config)) (*R
 		Client:          client,
 	}, nil
 }
+
+func (r *Resource) Contents(ctx context.Context, prefix string) ([]File, error) {
+	contents := make([]File, 0)
+
+	opts := minio.ListObjectsOptions{
+		Recursive: true,
+		Prefix:    prefix,
+	}
+	for objInfo := range r.Client.ListObjects(ctx, r.BucketName, opts) {
```

Comment:
What happens if the context gets cancelled? Don't we have to check `objInfo.Err`? :thinking: